### ISSUE TITLE
monotonic loss

### ISSUE THREAD

**Issue by aisha-lawal** [aisha-lawal/inrmorph#9]
for this we take the $\frac{\partial|J|}{\partial t}$ this would lead to us having a resulting tensor of shape 4, 8, 12, 12, 12 (time, batch and spatial dims). we compute monotonicity such that: sum of all positive and negative voxelwise derivatives across time points we then pick the minimum between the sum of all positive and negative voxelwise derivatives across time points. 

```
positive_sum = torch.relu(voxelwise_derivatives).sum(dim=0)

negative_sum = torch.relu(-voxelwise_derivatives).sum(dim=0) 

mono_loss = torch.min(positive_sum, negative_sum).sum()
```
 toy example as follows

![image](https://github.com/user-attachments/assets/4758a0a5-b695-44d0-b990-61902541c29e)
